src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -21,49 +24,129 @@ public class SortCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
         + ": Sort the list of contacts displayed by certain parameter(s).\n"
+        + "Insert a ! after a / to sort in reverse.\n"
```

Comment:
ArgumentTokenizer detects arguments in the form of `" prefixVALUE"`. (For example `" t/friend"`)
If I go with `!/n` I would either have to 
1. make a new Tokenizer that detects `" !prefixVALUE"` or 
2. pass in each prefix twice, one with `!`, one without

Instead, I changed the command so that I can extract out `!` from the argument value.